master/getting-started/docker/installation/requirements.md

```diff
@@ -24,7 +24,9 @@ To use Calico as a Docker network plugin, the Docker daemon must be configured
 with a cluster store.  If using etcd as a cluster store,
 configure the `cluster-store` on the Docker daemon to `etcd://<ETCD_IP>:<ETCD_PORT>`,
 replacing `<ETCD IP>` and <ETCD_PORT> with the appropriate address and client
-port for your etcd cluster.
+port for your etcd cluster.  If your etcd is configured with TLS then you must
+also add [configuration to the Docker daemon][daemon-cert-config] to allow
```

Comment:
s/also add configuration to the Docker daemon/also configure the docker daemon/g